mitsurunner.yaml

```diff
@@ -137,14 +159,14 @@ sensor:
                 break;
 
             case ST_START_DEFROSTING:
-                // Check that defrosting is really started.
-                if (temperature_delta < 0.0) {
-                    id(enter_DefrostingStarted).execute();
-                }
                 // Timeout to go back to the idle state, if defrosting has not been started.
-                else if (id(G_state_time_passed)) {
+                if (id(G_state_time_passed)) {
```

Comment:
Just switched the order? Ok for me.

Reply:
Small nuance. Would be better to switch state instead of action, when both conditions are possible to happen there.